main.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import os.path
+from argparse import Namespace
+
+ERR_MSG = "Error message: Invalid input. Make sure the input file is a valid csv file."
+
+
+def get_args() -> Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input_file", type=str, required=True)
+    parser.add_argument("-o", "--output_file", type=str, required=True)
+    parser.add_argument("-r", "--row_limit", type=int, required=True)
+    return parser.parse_args()
+
+
+def is_input_valid(input_file: str, row_limit: int) -> bool:
```

Comment:
This function does too many things. 
Its name implies that its sole purpose is to check if the input is valid, but in the implementation it also opens a file, checks the number of rows...
Think about what is the responsibility of this function and make sure it only performs one task.
See [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single-responsibility_principle).